packages/shared-base/src/Messages/Mask.ts

```diff
@@ -3,6 +3,7 @@ import type { SerializableTypedMessages } from '@masknet/typed-message'
 import type { ProfileIdentifier, PersonaIdentifier } from '../Identifier'
 import type { RelationFavor } from '../Persona/type'
 import type { EnhanceableSite, ExtensionSite } from '../Site'
+import type { ChainId } from '@masknet/web3-shared-evm'
```

Comment:
circular dependency? shared-base should not refer to it. maybe you should move the definition of ChainId to this module.